lib/smart_answer_flows/am-i-getting-minimum-wage/questions/current_paid_for_work_outside_shift.govspeak.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for :title do %>
+  Does your employer pay you for any additional time you work outside of your shift?
+<% end %>
+
+<% options(
+  "yes": "Yes",
+  "no": "No"
+) %>
```

Comment:
You're missing a newline at the end of this file (and some other files as well).